routes/scheduleStack.js

```diff
@@ -16,6 +19,24 @@ const screens = {
             headerShown: false,
         }
     },
+    DoubleSearch: {
```

Comment:
Same comment as the one added in `NearbyInterestStack.js`

Reply:
Got it!